### PR TITLE
 [fix bug 1129162] Respect blank value of model field. 

### PR DESCRIPTION
**Pull request by glogiotatidis**
Ideally I want to move the custom label code into a special admin widget if possible. I'll do a widget research on how to make TinyMCE on demand and maybe I figure out a way to fix this too. 

Meanwhile to unblock this, here's a quick fix.

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@jgmize r?

File: masterfirefoxos/base/models.py [hunk 1]
```diff
@@ -70,7 +70,8 @@ class MediaFileInline(FeinCMSInline):
     def formfield_for_foreignkey(self, db_field, request, **kwargs):
         if isinstance(db_field, MediaFileForeignKey):
             return CustomMediaFileTypeChoiceField(
-                MediaFile.objects.filter(type='image', categories__title='en'), **kwargs)
+                MediaFile.objects.filter(type='image', categories__title='en'),
+                required=not db_field.blank)
```
Comment (jgmize) [inline masterfirefoxos/base/models.py]: Shouldn't we include `**kwargs` as well?

**Comment by jgmize** [comment]
r+wc

File: masterfirefoxos/base/models.py (hunk 1 above)
Comment (glogiotatidis) [inline masterfirefoxos/base/models.py]: kwargs is always empty, but i'll include this just to be on the safe side

**Comment by glogiotatidis** [comment]
thanks @jgmize

**Pull request merged**